src/Util/VerboseLogging.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc;
+
+namespace sodoff.Utils;
+
+public class VerboseLogging {
+    public static void log() {
+        Console.WriteLine("[INFO] Server is loading Assets");
```

Comment:
I don't think this is accurate.  Where this is called, the server is _not_ loading assets.

Reply:
i removed  that i need add a logger for that
tomorrow i have more time for that
i had ideas to read from the api